lib/makePizza.js

```diff
@@ -0,0 +1,54 @@
+//Code simplified and used from node module 'google-images'
+//https://github.com/vdemedes/node-google-images
+
+var request = require("request");
+
+module.exports = {
+  //Convert pizza ingredients into a string
+  convertToPizzaString: function(crust, cheese, toppings) {
+    var pizzaString = crust + " pizza ";
+    pizzaString += "with " + cheese + " cheese ";
+    pizzaString += "and " + toppings;
+    return pizzaString;
+  },
+  //Search for an image via google
+  search: function(query, originalRes, callback) {
+    //Use request module to get img
```

Comment:
This line to about line 32 are the lines that I took from the 'google-images' module. Re reading my code, I realize that's not clear.